packages/ui/src/components/MenuV2/Item.tsx

```diff
@@ -0,0 +1,152 @@
+import type { Theme } from '@emotion/react'
+import styled from '@emotion/styled'
+import type { MouseEventHandler, ReactNode, Ref } from 'react'
+import { forwardRef } from 'react'
+import { Tooltip } from '../Tooltip'
+
+type MenuItemSentiment = 'neutral' | 'danger'
+
+const itemCoreStyle = ({
+  theme,
+  borderless,
+  sentiment,
+  disabled,
+}: {
+  theme: Theme
+  borderless: boolean
+  sentiment: MenuItemSentiment
+  disabled: boolean
+}) => `
+  display: inline-block;
+  font-size: ${theme.typography.bodySmall.fontSize};
+  line-height: ${theme.typography.bodySmall.lineHeight};
+  font-weight: inherit;
+  padding: ${`${theme.space['0.5']} ${theme.space['1']}`};
+  border: none;
+  ${
+    borderless ? '' : `border-bottom: 1px solid ${theme.colors.neutral.border};`
+  }
+  cursor: pointer;
+  min-width: 110px;
+
+  color: ${theme.colors[sentiment][disabled ? 'textDisabled' : 'text']};
+  svg {
+    fill: ${theme.colors[sentiment][disabled ? 'textDisabled' : 'text']};
+  }
+
+  ${
+    disabled
+      ? `
+        cursor: not-allowed;
+      `
+      : `
+          &:hover,
+          &:focus {
+            color: ${theme.colors[sentiment].textHover};
+            svg {
+              fill: ${theme.colors[sentiment].textHover};
+            }
+          }`
+  }
+  `
+
+const StyledItem = styled('button', {
+  shouldForwardProp: prop => !['borderless', 'sentiment'].includes(prop),
+})<{
+  borderless: boolean
+  disabled: boolean
+  sentiment: MenuItemSentiment
+}>`
+  ${({ theme, borderless, sentiment, disabled }) =>
+    itemCoreStyle({ theme, borderless, sentiment, disabled })}
+  background: none;
+`
+
+const StyledLinkItem = styled('a', {
+  shouldForwardProp: prop => !['borderless', 'sentiment'].includes(prop),
+})<{
+  borderless: boolean
+  disabled: boolean
+  sentiment: MenuItemSentiment
+}>`
+  ${({ theme, borderless, sentiment, disabled }) =>
+    itemCoreStyle({ theme, borderless, sentiment, disabled })}
+  text-decoration: none;
+
+  &:focus {
+    text-decoration: none;
+  }
+`
+
+type ItemProps = {
+  href?: string | undefined
+  disabled?: boolean | undefined
+  tooltip?: string | undefined
+  className?: string | undefined
+  children: ReactNode
+  onClick?: MouseEventHandler<HTMLElement> | undefined
+  borderless?: boolean
+  sentiment?: MenuItemSentiment
+  'data-testid'?: string
+}
+
+const Item = forwardRef<HTMLElement, ItemProps>(
+  (
+    {
+      borderless = false,
+      disabled = false,
+      onClick,
+      sentiment = 'neutral',
+      href,
+      children,
+      tooltip,
+      className,
+      'data-testid': dataTestId,
+    },
+    ref,
+  ) => {
+    if (href && !disabled) {
+      return (
+        <Tooltip text={tooltip}>
+          <StyledLinkItem
+            borderless
```

Comment:
Do we always want that Link Items are borderless?

Reply:
It's only borderless when there is an href, similar to Menu so I guess it's fine.